src/nvim/charset.c

```diff
@@ -1454,6 +1454,20 @@ char_u* skipdigits(char_u *q)
   return p;
 }
 
+/// skip over binary digits
+///
+/// @param q
+///
+/// @return Pointer to the character after the skipped digits.
+char_u* skipbin(char_u *q)
+{
+    char_u	*p = q;
+
+    while (ascii_isbdigit(*p))	/* skip to next non-digit */
+	++p;
```

Comment:
According to the style guide increment/decrement statements should use post-increments. This rule is only not applicable if increment/decrement is a part of the expression.